modules/plugins/grid.js

```diff
@@ -1,10 +1,10 @@
 /* @flow */
 
 function isSimplePositionValue(value: any) {
-  return typeof value === 'number' && !isNaN(value)
+  return typeof value === 'number' && !Number.isNaN(value)
```

Comment:
Actually, I wonder if `Number.isFinite` is what you want here.

Also, I noticed that IE does not support either, so maybe what you originally had would be better in the end?

Reply:
@lencioni I went back to `isNaN` and removed the use of `Set` because I don't think IE11 supports constructing one [from an iterable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#Browser_compatibility).